src/main.rs

```diff
@@ -8,26 +9,36 @@ mod config;
 mod rules;
 
 fn main() {
-    for arg in env::args().skip(1) {
-        check_file(&arg);
+    let mut args: Vec<String> = env::args().skip(1).collect();
+    if args.is_empty() {
+        args = vec![String::from(".")]
+    }
+    for arg in args.iter() {
```

Comment:
Nit: `.iter()` should not be needed here. Rust will automatically accept anything that implements `IntoIterator` for `for` loops, which `Vec` does.